websites/S/Scratch/dist/metadata.json

```
@@ -11,7 +11,7 @@
 	},
 	"url": "scratch.mit.edu",
 	"regExp": "([a-z0-9-]+[.])*(scratch|scratchfoundation|scratch-wiki|donationpay|scratched[.]gse[.]harvard|scratchjr)[.](mit[.]edu|org|info|edu)[/]",
-	"version": "1.2.12",
+	"version": "1.2.13",
```

You are adding quite a bit of functionality, probably should bump minor instead
```suggestion
	"version": "1.3.0",
```